doc/architecture/decisions/0005-rooms-and-beds-for-both-cas-services.md

```diff
@@ -0,0 +1,106 @@
+# 5. rooms-and-beds-for-both-cas-services
+
+Date: 2022-10-24
+
+## Status
+
+Accepted
+
+## Context
+
+The CAS1 team have a longer term vision of the data layer needed for their
+property management service[1]. At the time of writing this part of the service
+is partially complete and called "mini-manage". It includes Premises but not
+Rooms or Beds.
+
+CAS1 have introduced a temporary bed concept by allowing users to store a
+`bed_count` on Premises directly. It was imagined that this would eventually
+change over.
+
+CAS3 are looking to extend mini-manage now in way that works for CAS3 but is
+also compatible for CAS1. Both teams built a shared outstanding going into this
+work:
+
+- Both CAS1 and CAS3 need to represent rooms in some fashion and associate them
+  with bookings in future
+- CAS1 will have rooms with one or multiple beds
+- CAS3 will have one bed per room, it is highly unlikely but still possible
+  that in future this might change
+- Physical spaces such as Premises and Rooms look to have different
+  characteristics for CAS1 and CAS3. For example, CAS3 needs to specify whether
+  a room has a shared kitchen but CAS1 may not, however CAS1 might need to know
+  if it has an en suite.
+
+[1] <https://dbdiagram.io/d/6200fa0285022f4ee54d7569>
+
+## Decision
+
+- We will attempt to build and share the core room and bed resources
+- We will separate out the storage of unique characteristics from
+  rooms and beds in a way that supports future flexibility for both services
+- We imagine[2] we can create a new Characteristics table where records can be
+  created to represent boolean attributes. These could be associated with
+  premises or rooms via two further join tables:
+
+         ,---------------------------------------.
+         |Characteristic                         |
+         |---------------------------------------|
+         |*id: integer <<autonumber>>            |
+         |--                                     |
+         |*name: string                          |
+         |*service_scope: string <<default: "*">>|
+         |*model_scope: string <<default: "*">>  |
```

Comment:
I'm curious about how you would use these scopes? 

Is it to display a list of characteristics which could (but have not yet) been applied to a CAS3 room? The list of characteristics which have been used can readily be surfaced with a join, so perhaps this is anticipating the need to apply previously unused characteristics through the frontend UI?